artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -272,6 +272,23 @@ public void onNotification(final Notification notification) {
 
                queueInfos.put(clusterName, info);
 
+               //Global queue creation fix
+               Binding binding = getBinding(routingName);
+               
+               AddressSettings addressSettings = addressSettingsRepository.getMatch(address.toString());
+
+               boolean clusteredQueues = addressSettings.getClusteredQueues();
```

Comment:
should this not be a value that's configurable at the queue level, e.g. under an address you may want one queue not clustered and another clustered. it shouldn't be an all or nothing for all queues, e.g. address setting should be a default, but the actual value should be at the queue level